src/main/resources/default.properties

```diff
@@ -6,3 +6,6 @@ JDBC_URL=127.0.0.1
 DATABASE_URL="test_url"
 DATABASE_PASSWORD="test_password"
 DATABASE_USERNAME=test_username
+BACKUP_FREQUENCY="daily"
```

Comment:
There should be an option to disable database backups. Database backups give a 100% utilisation for the disk a lot of the times. It is for this reason that a lot of big websites only backup manually. Also, we should add a yearly option.